help_to_heat/middleware.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.core.handlers.wsgi import WSGIRequest
+
+logger = logging.getLogger(__name__)
+requests_logger = logging.getLogger("help_to_heat.requests")
+
+
+class ExceptionMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        # pass requests down the stack without edits
+        return self.get_response(request)
+
+    def process_exception(self, _request, exception):
+        # print out to AWS logs
+        logger.exception(exception)
+        return None
+
+
+class RequestLoggingMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request: WSGIRequest):
+        # when required uncomment for request logging
```

Comment:
Feels like it would be better to comment out the middleware in `settings.py`?

Reply:
sure; the thinking was that it would be less of a change later on to just uncomment the logging line, though I suppose means we would be introducing more into the stack now